<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="StatusLabel">
          <property name="text">
           <string>ShutDown</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="RoomNumLabel">
          <property name="text">
           <string>RoomNum:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="RoomNumLineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="UserIDLabel">
          <property name="text">
           <string>UserID:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="UserIDLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="LoginBtn">
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="RoomTempLabel">
            <property name="text">
             <string>Room Temperature:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="HighestTemperature">
            <property name="text">
             <string>Enable Highedst:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="LowestTemperature">
            <property name="text">
             <string>Enable Lowest:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="RealTimeTemperatureLabel">
            <property name="text">
             <string>25 Celsius</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="HighestTemperatureLabel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="LowestTemperatureLabel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="WorkMode">
            <property name="text">
             <string>WorkMode:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Consumption">
            <property name="text">
             <string>Consumption:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Fee">
            <property name="text">
             <string>Fee:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="WorkModeLabel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="ConsumptionLabel">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="FeeLabel">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="TargetTempLabel">
            <property name="text">
             <string>Target Temperatureï¼š</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TargetTemperatureLabel">
            <property name="text">
             <string>25</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="UpTemperatureBtn">
            <property name="text">
             <string>Up</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="DownTemperatureBtn">
            <property name="text">
             <string>Down</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="WindVelocity">
            <property name="text">
             <string>Wind Velocity:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="WindVelocityLabel">
            <property name="text">
             <string>Low</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="HighVelocityBtn">
            <property name="text">
             <string>High</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="MidVelocityBtn">
            <property name="text">
             <string>Mid</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="LowVelocityBtn">
            <property name="text">
             <string>Low</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>746</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
